pycommon/report2idea.py

```diff
@@ -215,6 +297,11 @@ def Run(module_name, module_desc, req_type, req_format, conv_func, arg_parser =
             # Pass the input record to conversion function to create IDEA message
             idea = conv_func(rec, args)
 
+            # Sanity check of timestamps
+            if not check_valid_timestamps(idea):
+                print("Invalid timestamps in skipped message: {0}".format(idea))
+                continue
```

Comment:
This will crash if `idea is None`. Move this block below the test for `None` (and maybe add a test with a message which can't be converted so `conv_func` returns `None`).

Reply:
fixed in c3d5063, thanks for reporting this bug